Xamarin.Forms.Core/SwipeView.cs

```diff
@@ -0,0 +1,742 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Collections.Specialized;
+using System.ComponentModel;
+using System.Linq;
+using System.Windows.Input;
+using Xamarin.Forms.Platform;
+
+namespace Xamarin.Forms
+{
+	[ContentProperty("Content")]
+	[RenderWith(typeof(_SwipeViewRenderer))]
+	public class SwipeView : ContentView, ISwipeViewController, IElementConfiguration<SwipeView>
+	{
+		const double SwipeThreshold = 250; 
+  		const double SwipeItemWidth = 80;
+		const uint SwipeAnimationDuration = 100;
+
+		bool _isTouchDown;
+		Point _initialPoint;
+		SwipeDirection _swipeDirection;
+		double _swipeOffset;
+		readonly Grid _content;
+		View _view;
+		readonly IList<Element> _gridChild;
+		readonly ReadOnlyCollection<Element> _logicalChildren;
+		readonly Lazy<PlatformConfigurationRegistry<SwipeView>> _platformConfigurationRegistry;
+
+		public SwipeView()
+		{
+			IsClippedToBounds = true;
+
+			_content = new Grid();
+			_view = new View();
+
+			_gridChild = new List<Element>();
+
+			CompressedLayout.SetIsHeadless(_content, true);
```

Comment:
COURAGE

Reply:
:satisfied: This implementation has already changed and is not used.